src/index.ts

```diff
@@ -46,8 +47,33 @@ if (runningLocally) {
 }
 const firestore = firestoreAdmin.firestore();
 
+// setup discord.js commands
+client.on('ready', () => {
+  for (const [guildId] of client.guilds.cache) {
+    if (!client.user) {
+      console.error(client.user);
+      continue;
+    }
+
+    deployCommands(client.user.id, guildId, process.env.TOKEN!)
+  }
+});
+
+// listen for discord.js command
+client.on('interactionCreate', async interaction => {
+	if (!interaction.isCommand()) return;
+
+	try {
+    const commandFile = require(`./commands/${interaction.commandName}`); // Loads up the command based on file name
```

Comment:
Also weird formatting? Or is this just weird github thing?

Reply:
+1

Reply:
Seems to be only in the browser, not seeing this locally